Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -14,6 +14,7 @@ private let BookmarkSeparatorCellIdentifier = "BookmarkSeparatorCellIdentifier"
 
 private struct BookmarksPanelUX {
     static let FolderIconSize = CGSize(width: 20, height: 20)
+    static let RowFlashDelayInMilliseconds = 400
```

Comment:
nit,  I prefer: ` rowFlashDelay: TimeInterval = 0.4`, as TimeInterval is well-defined as seconds. Then you don't need to wrap it in `.milliseconds`. Up to you though